src/main/java/de/unijena/cheminf/mortar/model/fragmentation/algorithm/ScaffoldGeneratorFragmenter.java

```diff
@@ -340,7 +342,8 @@ public static enum FragmentationTypeOption {
      */
     public ScaffoldGeneratorFragmenter() {
         this.scaffoldGeneratorInstance = new ScaffoldGenerator();
-        this.settingNameTooltipTextMap = new HashMap(16, 0.9f);
+        int tmpInitialCapacityForSettingNameTooltipTextMap = CollectionUtil.calculateInitialHashCollectionCapacity(16, BasicDefinitions.DEFAULT_HASH_COLLECTION_LOAD_FACTOR);
+        this.settingNameTooltipTextMap = new HashMap(tmpInitialCapacityForSettingNameTooltipTextMap, BasicDefinitions.DEFAULT_HASH_COLLECTION_LOAD_FACTOR);
         this.fragmentSaturationSetting = new SimpleEnumConstantNameProperty(this, "Fragment saturation setting",
```

Comment:
see comment above

Reply:
Here, I just adjusted what Julian did. I have introduced a variable above with the number of settings (10 in this case, not 16...).